src/System.Linq.Queryable/src/System.Linq.Queryable.csproj

```diff
@@ -32,5 +32,12 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Linq\src\System.Linq.csproj">
```

Comment:
Ah. I've got it. Don't have how to not have to manually overwrite packages though (is there somewhere that's good for these newb-to-project questions?).